pandas/core/common.py

```diff
@@ -473,3 +475,14 @@ def f(x):
         f = mapper
 
     return f
+
+
+def convert_to_list_like(values):
+    if hasattr(values, "dtype"):
```

Comment:
can u type and add a doc string
make these elif 

see where else we can u this